src/wranglerAction.ts

```diff
@@ -0,0 +1,456 @@
+import {
+	debug,
+	getMultilineInput,
+	endGroup as originalEndGroup,
+	error as originalError,
+	info as originalInfo,
+	startGroup as originalStartGroup,
+	setFailed,
+	setOutput,
+} from "@actions/core";
+import { getExecOutput } from "@actions/exec";
+import semverEq from "semver/functions/eq";
+import { z } from "zod";
+import { exec, execShell } from "./exec";
+import { PackageManager } from "./packageManagers";
+import { semverCompare } from "./utils";
+import { getDetailedPagesDeployOutput } from "./wranglerArtifactManager";
+
+export type WranglerActionConfig = z.infer<typeof wranglerActionConfig>;
```

Comment:
Now that we're defining a type for config, we should use it when instantiating `config` in `index.ts` here https://github.com/cloudflare/wrangler-action/pull/322/files#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80L40

```tsx
/**
 * A configuration object that contains all the inputs & immutable state for the action.
 */
const config: WranglerActionConfig = {
	WRANGLER_VERSION: getInput("wranglerVersion") || DEFAULT_WRANGLER_VERSION,
	didUserProvideWranglerVersion: Boolean(getInput("wranglerVersion")),
	secrets: getMultilineInput("secrets"),
	workingDirectory: checkWorkingDirectory(getInput("workingDirectory")),
	CLOUDFLARE_API_TOKEN: getInput("apiToken"),
	CLOUDFLARE_ACCOUNT_ID: getInput("accountId"),
	ENVIRONMENT: getInput("environment"),
	VARS: getMultilineInput("vars"),
	COMMANDS: getMultilineInput("command"),
	QUIET_MODE: getBooleanInput("quiet"),
	PACKAGE_MANAGER: getInput("packageManager"),
	WRANGLER_OUTPUT_DIR: `${join(tmpdir(), "wranglerArtifacts")}`,
} ;
```

Reply:
Good point! Thanks!